.github/workflows/cibuildwheel.yml

```diff
@@ -10,13 +10,89 @@ on:
       - v*
 
 jobs:
-  build_wheels:
+  build_linux_36_wheels:
```

Comment:
@janaknat Do I interpret this action correctly that it only builds wheels for 36 on the 0.17 branch? Would you like us to push out a final 0.17.x release, after which this job can be deleted? Or am I misinterpreting this?

Reply:
I went by what was already present in the cibuildwheel.yml file for x86. I just checked PyPI and it looks like python 3.6 didn't have any wheels. Then I would go ahead and only release 3.7, 3.8 and 3.9 wheels. I can update the PR to remove the config for python 3.6.